CHANGELOG.md

```diff
@@ -5,12 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.1.0]
```

Comment:
Technically speaking, the current change should be a separate entry in the CHANGELOG

Reply:
Perhaps :)

I would've amended your commit but wasn't able to due to the fork.

Good eye tho!